tests/test_with_http.py

```diff
@@ -115,6 +131,12 @@ def test_deserialization(self):
         scitoken = scitokens.SciToken.deserialize(serialized_token, insecure=True)
 
         self.assertIsInstance(scitoken, scitokens.SciToken)
+        
```

Comment:
Codacy complains about trailing whitespace here (I think maybe your editor is setup to behave like this?).  Once that's done, I'm happy!